sync_ci/pkg/parser/ci_log.go

```diff
@@ -101,6 +101,17 @@ func refineParseRes(res map[string][]string) {
 			delete(res, v)
 		}
 	}
+
+	// The input will only fall on one category according to ranking
```

Comment:
I'm a bit confused. Wouldn't there be some possibility for one job having two sorts of errors?